### PR TITLE
Proposal modal

### PR DESCRIPTION
**Pull request by brossetti1**
## GitHub Issue

https://github.com/HausDAO/daohaus-monorepo-deprecated/issues/1003

## Changes

- [X] add tabs component (not to comfortable with how its come out, need some feedback)
- [X] split proposals into two tabs, basic and advanced

## Packages Added

N/A

## Checks

Screen Shot 2022-11-22 at 11 07 35 AM
Screen Shot 2022-11-22 at 11 07 39 AM

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for _any impacted package_ and check for any errors before the PR)

### CODE REVIEW

File: apps/admin/src/pages/Proposals.tsx [hunk 1]
```diff
@@ -102,6 +101,11 @@ export function Proposals() {
     });
   };
 
+  useEffect(() => {
+    filterProposals();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
```
Comment (brossetti1) [inline apps/admin/src/pages/Proposals.tsx:108]: simply moving this

File: apps/admin/src/components/NewProposalList.tsx [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
-import { Bold, border, DataSm, Link, ParMd, Theme } from '@daohaus/ui';
+import {
+  Bold,
+  border,
+  DataSm,
+  Link,
+  ParMd,
+  Tabs,
+  Theme,
+  Divider,
```
Comment (skuhlmann) [inline apps/admin/src/components/NewProposalList.tsx:12]: remove this import

File: apps/admin/src/pages/Proposals.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { MouseEvent, useEffect, useMemo, useState } from 'react';
+import { MouseEvent, useCallback, useEffect, useMemo, useState } from 'react';
```
Comment (skuhlmann) [inline apps/admin/src/pages/Proposals.tsx:1]: cleanup

**Review by skuhlmann — APPROVED** [review]
looks good to me with some cleanup

**Pull request merged**